RadixWallet/Clients/AccountLockersClient/AccountLockersClient+Interface.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+// MARK: - AccountLockersClient
+public struct AccountLockersClient: DependencyKey, Sendable {
+	public let startMonitoring: StartMonitoring
+
+	/// Async sequence with the AccountLockerClaims for a given Account
+	public let accountClaims: AccountClaims
+
+	/// Async sequence with the set of dapps that have at least one AccountLockerClaim associated to it.
+	public let dappsWithClaims: DappsWithClaims
+}
+
+// MARK: AccountLockersClient.StartMonitoring
+extension AccountLockersClient {
+	public typealias StartMonitoring = @Sendable () async throws -> Void
+	public typealias AccountClaims = @Sendable (AccountAddress) async -> AnyAsyncSequence<[AccountLockerClaims]>
+	public typealias DappsWithClaims = @Sendable () async -> AnyAsyncSequence<[String]>
+}
+
+// MARK: - AccountLockerClaims
+/// A struct holding the pending claims for a given locker address & account address.
+public struct AccountLockerClaims: Sendable, Hashable, Codable {
+	let lockerAddress: String
+	let accountAddress: String
+	let dappDefinitionAddress: String
+	let dappName: String?
+	let lastTouchedAtStateVersion: Int64
```

Comment:
you mean to convert the API's `lastTouchedAtStateVersion` into something like `timeIntervalSinceReferenceDate`? I would leave it as is, without modifying it and risking to have the value changed on any scenario, since we only use it to see if it has changed (we don't care about how much time has passed since we last checked, or anything)

Reply:
No, I mean that we do have `Epoch` type defined. But it is different from StateVersion from the GW perspective. I would then define a StateVersion typealias at least